File: backend/micro_epoweredmove_reviews_reservations/src/main/java/com/kosdiam/epoweredmove/models/enums/CurrentType.java
package com.kosdiam.epoweredmove.models.enums;

public enum CurrentType {
    AC("AC"),
    DC("DC");

    private String currentType;
    private CurrentType(String currentType){
        this.currentType = currentType;
    }

    @Override
    public String toString(){
        return currentType;
    }
}

File: frontend/E-PoweredMoveAndroidApp/app/src/main/java/com/kosdiam/epoweredmove/models/enums/CurrentType.kt
package com.kosdiam.epoweredmove.models.enums

import com.kosdiam.epoweredmove.R
import com.kosdiam.epoweredmove.models.interfaces.IEnumName

enum class CurrentType (override val stringResource: Int) : IEnumName {
    AC(R.string.current_type_ac),
    DC(R.string.current_type_dc);
}